CMSgov/design-system
# [WNMGDS-843] updated spinner markup for improved accessibility

## Summary
Overview of changes


## How to test
Navigate to spinner component at demo site: http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-843/spinner-update. If you have the axe browser extension installed, confirm that the error "Spinner component doesn't have an accessible name" is not longer present.

Compare that spinner to what is on https://design.cms.gov/components/spinner/. Confirm that stylistically, nothing has changed.

Run `yarn test` on branch and confirm that all tests pass
